src/App.js

```diff
@@ -1,8 +1,20 @@
 import './App.css';
+import { Routes, Route } from 'react-router-dom';
+
+import FishProfiles from './components/FishProfiles/FishProfiles';
+import NavBar from './components/Nav/NavBar';
+import Detail from './components/Detail/Detail';
 
```

Comment:
- [ ] Good job so far but your app is running with many errors, I noticed that there were several `modules` not found errors in your React application. The errors appear to be related to missing dependencies such as `react-router-dom`, , `react-redux`, and `react-icons/fa`. These dependencies are necessary for certain components in your application, such as `Detail.js`,`FishProfiles.js`, and `NavBar.js`. kindly fix this error :point_down:  then ask for the second opinion when you are ready :wink: 
![Screenshot from 2023-05-06 13-06-37](https://user-images.githubusercontent.com/88894525/236620806-30e7270f-3fea-4940-b452-0be3b9ca8f1c.png)